intercept_openai_chat_streaming.go

```diff
@@ -161,7 +165,7 @@ func (i *OpenAIStreamingChatInterception) ProcessRequest(w http.ResponseWriter,
 				MsgID:          processor.getMsgID(),
 				Input:          calculateActualInputTokenUsage(lastUsage),
 				Output:         lastUsage.CompletionTokens,
-				Metadata: Metadata{
+				ExtraTokenTypes: map[string]int64{
```

Comment:
Actually, I'm not convinced about the property name indicating that these metrics are extra. IMHO let's preserve the original name, so `ProviderTokenDetails`, but I will not insist on renaming 👍

Reply:
They're "extra" because they're over and above the common input/output tokens. I'd like to keep this naming.